### PR TITLE
Add TogglDesktop.app :latest

### PR DESCRIPTION
**Pull request by smukkejohan**

### CODE REVIEW

File: Casks/toggl.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'toggl' do
+  version :latest
+  sha256 '89cbb9d701e5bc612542cdfd35ca182e93ff5c81ddc21c183b2cb867fad84937'
```
Comment (fanquake) [inline Casks/toggl.rb]: if your using version :latest you should be using sha256 :no_check

File: Casks/toggl.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'toggl' do
+  version :no_check
```
Comment (vitorgalvao) [inline Casks/toggl.rb]: This is wrong. It should be `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/master/Casks/acorn.rb).

**Comment by vitorgalvao** [comment]
Thank you, but [already present](https://github.com/caskroom/homebrew-cask/blob/master/Casks/toggldesktop.rb). You need to follow the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_token_reference.md) to name casks. It helps filter duplicates.

**Comment by mrbichel** [comment]
Oh sorry. I somehow missed that. I did a search though. 



-
Johan Bichel Lindegaard
http://johan.cc
public@johan.ccDK +45 23600463


US +1 9292489146


SKYPE mr.bichel

**Pull request closed without merging**